server_addon/maya/server/settings/publishers.py

```diff
@@ -309,6 +386,166 @@ class ExtractAlembicModel(BaseSettingsModel):
         description="List of attribute prefixes for attributes that will be "
                     "included in the alembic export.",
     )
+    dataFormat: str = SettingsField(
+        enum_resolver=extract_alembic_data_format_enum,
+        title="Data Format",
+        description="The data format to use to write the file."
+    )
+    dontSkipUnwrittenFrames: bool = SettingsField(
+        title="Dont Skip Unwritten Frames",
+        description=(
+            "When evaluating multiple translate jobs, this decides whether to "
+            "evaluate frames between jobs when there is a gap in their frame "
+            "ranges."
+        )
+    )
+    eulerFilter: bool = SettingsField(
+        title="Euler Filter",
+        description="Apply Euler filter while sampling rotations."
+    )
+    melPerFrameCallback: str = SettingsField(
+        title="Mel Per Frame Callback",
+        description=(
+            "When each frame (and the static frame) is evaluated the string "
+            "specified is evaluated as a Mel command."
+        )
+    )
+    melPostFrameCallback: str = SettingsField(
+        title="Mel Post Frame Callback",
+        description=(
+            "When the translation has finished the string specified is "
+            "evaluated as a Mel command."
+        )
+    )
+    noNormals: bool = SettingsField(
+        title="No Normals",
+        description=(
+            "Present normal data for Alembic poly meshes will not be written."
+        )
+    )
+    preRoll: bool = SettingsField(
+        title="Pre Roll",
+        description=(
+            "When enabled, the pre roll start frame is used to pre roll the "
+            "export for."
+        )
+    )
```

Comment:
```suggestion
    preRoll: bool = SettingsField(
        title="Pre Roll",
        description=(
            "When enabled, the pre roll start frame is used to pre roll the "
            "When enabled, the pre roll start frame is used to being the "
            "evaluation of the mesh. From the pre roll start frame to the "
            "alembic start frame, will not be written to disk. This can be "
            "used for simulation run up."
        )
    )
```